src/data_gradients/datasets/FolderProcessor.py

```diff
@@ -0,0 +1,97 @@
+import os
+import logging
+from typing import List, Optional, Tuple
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_IMG_EXTENSIONS = ("bmp", "dng", "jpeg", "jpg", "mpo", "png", "tif", "tiff", "webp", "pfm")
```

Comment:
ppm

Reply:
I updated the tuple with all the extensions officially supported by opencv
https://github.com/Deci-AI/data-gradients/pull/114/commits/3ae7a1bf43398ec354b820cae1b9c66e4cd0ec97

We were missing a lot of them, but `dng` and `pfm` are apparently not natively supported with opencv
There are workarounds but do we want to support it? (see https://stackoverflow.com/questions/29487925/opencv-how-to-read-pfm-files)